content/docs/contributing/guidelines.md

```diff
@@ -1,22 +1,23 @@
 ---
 title: Contribution Guidelines
 id: /docs/contributing/guidelines
-prev: null
-next: /docs/contributing/setting-up
+next: /content/docs/contributing/setting-up.md
+_template: doc
```

Comment:
Could we automate the application of a schema to a collection to provide easier migration to Tina Cloud?